src/backend_stub/index.js

```diff
@@ -18,6 +18,13 @@ module.exports = {
 		var app = express(http.createServer());
 		var appServer;
 		
+		app.use(config.staticBaseUrl, serveStatic(config.staticPath));
+		app.get('/', function (req, res) {
+			res.sendFile('index.html', {
```

Comment:
Dump question: Just the plain ```http://localhost:2015/static/index.html``` in the browser would not for the relative path reasons, right?

If I understand it right, that the relative URL path is the only reason, why not handle that on C++ side and keep this JavaScript code as universal as possible?

(The reason I'm confused is that ```'index.html'``` is eventually served by C++, so this looks like a somewhat cyclic dependency. I don't like what looks like a somewhat cyclic dependency. :-) )